servlet/src/main/java/com/linecorp/armeria/server/servlet/DefaultServletService.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.servlet;
+
+import javax.annotation.Nullable;
+import javax.servlet.http.HttpServlet;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpResponseWriter;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.ResponseHeaders;
+import com.linecorp.armeria.server.DefaultServiceRequestContext;
+import com.linecorp.armeria.server.HttpService;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+/**
+ * An {@link HttpService} which handles {@link HttpRequest} and forward to Servlet APIs,
+ * and write {@link HttpResponse} to client.
+ */
+public class DefaultServletService implements HttpService {
+    private static final Logger logger = LoggerFactory.getLogger(DefaultServletService.class);
+    private ServletHttpExchange servletHttpExchange;
```

Comment:
This must not be a field because of concurrency. How about just passing it when calling process method?

Reply:
I think it is a good idea 👍